project/Dependencies.scala

```diff
@@ -20,7 +20,7 @@ object Dependencies {
   val aeronVersion = "1.3.0"
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.11.11", "2.12.3"),
+    crossScalaVersions := Seq("2.12.4", "2.11.11"),
```

Comment:
Not sure that using 2.12 as default is good. Then there is a high risk that new things in 2.12 library is used and we don't find that until we run a later jenkins job with 2.11. What is the reason you want to make it default?